src/widget/wcoverart.cpp

```diff
@@ -19,17 +19,17 @@
 #include "widget/wskincolor.h"
 
 WCoverArt::WCoverArt(QWidget* parent,
-                     UserSettingsPointer pConfig,
-                     const QString& group,
-                     BaseTrackPlayer* pPlayer)
```

Comment:
I didn't do that intentionally this too, I think pre-commit made me do it in this way, is it correct or should it be like before?